src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/SceneFlameGraph.tsx

```diff
@@ -191,6 +192,14 @@ export class SceneFlameGraph extends SceneObjectBase<SceneFlameGraphState> {
       setRecordingRulesModalState({ isOpen: true, functionName });
     });
 
+    // Do not show AI button if the assistant integration is enabled to avoid having two AI buttons in the UI
+    // For debugging purposes and comparing both you can use localStorage flag grafana-pyroscope-app.forceShowAiButton
```

Comment:
This feels strange to add debugging flags that enabled functionality. Do we use this pattern elsewhere in Grafana? Perhaps we can remove if it's served its purpose? Technically, users can see the PR / code and enable it themselves which may lead to strange behaviour

Reply:
There's no harm if someone really wants to show both and uses the code from the PR. This is only to be able to compare both results. Don't think there's a similar pattern anywhere - this case is a bit special because we're replacing existing AI solution with a different one.